proposals/relaxed-simd/Overview.md

```diff
@@ -44,3 +44,71 @@ Example of some instructions we would like to add:
   and [meeting
   notes](https://github.com/WebAssembly/meetings/blob/master/main/2021/CG-03-16.md)
 - [SIMD proposal](https://github.com/WebAssembly/simd)
+
+## How to suggest a instructions
+
+1. [File an issue](https://github.com/WebAssembly/relaxed-simd/issues/new)
+2. In the body of the issue, copy the template below and answer the questions (an example of FMA has been given below):
+
+```
+1. What are the instructions being proposed?
+
+2. What are the semantics of these instructions?
+
+3. How will these instructions be implemented? Give examples on at 2 different architectures.
```

Comment:
Typo: `on at`. Besides, I think we should specifically ask for x86-64 and ARM64.

Reply:
sg, updated.